src/_test_/movieCounter.test.js

```diff
@@ -0,0 +1,33 @@
+/** @jest-environment jsdom */
```

Comment:
- [ ] Since the counters are supposed to count HTML elements in the DOM, the tests are also supposed to check if the counters are correctly working. Kindly use the example test suggested by the previous reviewer to achieve adequate results. 🧐️